Update game panel instructions (#42)
nanos-world/docs: README.md

## Installation

> Before running, make sure you have clonned all the files in git submodules (api, ...)

Reviewer: ```suggestion
Before running, make sure you have updated all submodules as well:
```
